FrameworkSystem/ConfigTemplate.cfg

```diff
@@ -135,6 +135,14 @@ Services
       Default = authenticated
     }  
   }
+  ComponentMonitoring
+  {
+    Port = 9190
+    Authorization
+    {
+      Default = authenticated
```

Comment:
Just thinking loud: I think that for this service we should define some restrictive authorization. Specifically, I would put Default = ServiceAdministrator